### ISSUE TITLE
actions on hyphenated options won't run

### ISSUE THREAD

**Issue by einarmagnus** [c4spar/deno-cliffy#135]
```typescript
import {Command} from "http://deno.land/x/cliffy@v0.16.0/command/mod.ts"

const cmd = new Command()
    .name("test")
    .version("0.1")
    .option("--testing <arg:string>", "single word flag", {
        action: options => {
            console.log("--testing", options);
        }
    })
    .option("--test-ing <arg:string>", "hyphenated word flag", {
        action: options => {
            console.log("--test-ing", options);
        }
    })
await cmd.parse(Deno.args);
```
```
PS C:\...> deno run .\test.ts --test-ing "hej!"
PS C:\...> deno run .\test.ts --testing "hej!" 
--testing { testing: "hej!" }
```
The first one never runs its action. This always happens if the option is hyphenated.

The problem is in `findActionFlag()` on https://github.com/c4spar/deno-cliffy/blob/ad779f46c817d9dd13a5c63031934d00d52282ca/command/command.ts#L1034

The problem is that the flag options that are hyphenated are changed to camelCase somewhere before handed over to `findActionFlag`, and then `getOption()` expects dash-case. If it is changed like this:

```typescript
protected findActionFlag(flags: O): IOption | undefined {
    const flagNames = Object.keys(flags);
    const dashCase = (id:string) => id.replace(/[A-Z]/g, c => `-${c.toLowerCase()}`);
    for (const flag of flagNames) {
      const option = this.getOption(dashCase(flag), true);  

      if (option?.action) {
        return option;
      }
    }

    return;
  }
```

then it works like it should. I'd send a pull request, but I feel like this is a hack that proves the problem, not a real solution.

**Comment by c4spar** [comment]
Hi @einarmagnus,

thx for finding this issue! Will look into it.

**Comment by c4spar** [comment]
Fixed in v0.17.1
